.github/analyst_guide_data_validation.md

```diff
@@ -0,0 +1,171 @@
+# Data validation
+
+This section covers the topic of **how to validate Workplace Analytics data**. Before you apply any functions to your data set and can start looking for new insights, it is recommended that you perform data validation. This best practice is applicable regardless of whether your aim is to explore the data, to establish a baseline, or to perform advanced analytics.
+
+## Why validate?
+
+There are several reasons why you should validate your Workplace Analytics data.
+
+1. There may be gaps, anomalies, or errors in the organizational data, such as missing data or excessive / insufficient granularity. This may require rectifying at source, or the resulting data should be interpreted differently, e.g. any biases caveated or accounted for. 
+2. Outliers may exist in Workplace Analytics data, and often for very legitimate reasons. For instance, collaboration hours for a particular week or employee may be significantly low due to public holidays or paid time off. If these outliers are not surfaced and addressed accordingly, the interpretation of the data may be incorrect.
+
+In a nutshell, it is good practice to have a comprehensive understanding of the data context and checks for common biases, errors, and anomalies prior to analysis, as otherwise we would risk the quality and the reliability of the analysis. 
+
+## Know what you are looking at
+
+Before you begin with data validation, it's helpful to know what data set you are looking at, which includes information such as: 
+
+  - What query type is loaded
+  - Number of unique employees in the dataset
+  - Date range of the dataset
+  - Organizational attributes int he dataset
+
+This can all be done with `check_query()`:
+
+```R
+library(wpa)
+check_query(sq_data)
+```
+
+The resulting output would look something like the following:
+
+```
+The data used is a Person Query.
+
+There are 1034 employees in this dataset.
+
+Date ranges from 2019-11-03 to 2020-01-26.
+
+There are 9 (estimated) HR attributes in the data:
+`Domain`, `FunctionType`, `LevelDesignation`, `Region`, `Organization`, `attainment`, `TimeZone`, `IsInternal`, `IsActive`
+
+There are 1034 active employees out of all in the dataset.
+
+Variable name check:
+
+`Collaboration_hours` is used instead of `Collaboration_hrs` in the data.
+
+`Instant_Message_hours` is used instead of `Instant_message_hours` in the data.
+```
+
+The below functions are also helpful for exploring your data:
+
+1. Get all column names, e.g. `names(sq_data)`
+2. Check object type, e.g. `class(sq_data$Date)`
+2. Get summary statistics, e.g. `summary(sq_data)`
+3. Compute number of unique values, e.g. `length(unique(sq_data$PersonId))`
+5. Get an overview of the data, e.g. `dplyr::glimpse(sq_data)`, or `skimr::skim(sq_data)`.
+
+Validating the structure of your data is just as important as validating the data itself. You may wish to check that your data is correctly imported into R if you observe certain anomalies, such as: 
```

Comment:
Note: just from reading that part - I am asking myself how to check the structure of my data. -> with view(data)?